pkgs/by-name/mk/mkbrr/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "mkbrr";
+  version = "1.11.0";
+
+  src = fetchFromGitHub {
+    owner = "autobrr";
+    repo = "mkbrr";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-NTkZKGoJqy7+f8dIIgLVRUQziD8QkcmoV2jEukXCqmM=";
+  };
+
+  vendorHash = "sha256-iczdGnJS70Ze5eZ2gnL511Ecy6Qjp+i4I749eIroRCM=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.Version=v${finalAttrs.version}"
+    "-X main.buildTime=unknown"
+  ];
+
+  doCheck = true;
```

```suggestion
  doCheck = true;
  
  doInstallCheck = true;
  nativeInstallCheckInputs = [
    versionCheckHook
  ];
```

To verify that version overriding is effective
https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook